### PR TITLE
CB-16609 Expose OpDB Agent service - correct CM and Knox service names

### PR DESCRIPTION
**Pull request by petersomogyi**
Fixed the OpDB Agent topology files. The service name in Knox is OPDB-AGENT and in CM it is called OPDB_AGENT.

### CODE REVIEW

File: cloudera/docker_images.yaml [hunk 1]
```diff
@@ -3,6 +3,7 @@ docker_images:
   cloudbreak-autoscale: hortonworks/cloudbreak-autoscale
   cloudbreak-datalake: hortonworks/cloudbreak-datalake
   cloudbreak-environment: hortonworks/cloudbreak-environment
+  cloudbreak-consumption: hortonworks/cloudbreak-consumption
```
Comment (lacikaaa) [inline cloudera/docker_images.yaml:6]: this must be dropped or the builds would fail

File: build.gradle [hunk 2]
```diff
@@ -161,8 +161,8 @@ subprojects {
     maxHeapSize = "1g"
     useJUnitPlatform()
     testlogger {
-      showSummary true
```
Comment (lacikaaa) [inline build.gradle:164]: why are you changing these?

**Review by lacikaaa — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 2 above)
Comment (bergerdenes) [reply build.gradle:164]: it was Ricsi's change, so I guess this PR might not be on top of the HEAD

Comment (petersomogyi) [reply build.gradle:164]: Let me rebase, when I moved the PR to CB-2.55.0 branch it picked up this commit that I haven't noticed.

**Review by bergerdenes — APPROVED** [review]

**Review by lacikaaa — APPROVED** [review]

**Pull request merged**